app/test/unit/infer-comparison-branch-test.ts

```diff
@@ -84,8 +78,14 @@ describe('inferComparisonBranch', () => {
     createTestBranch('default', '3', 'origin'),
     createTestBranch('head', '4', 'origin'),
     createTestBranch('upstream/base', '5', 'upstream'),
+    createTestBranch('fork', '6', 'origin'),
   ]
 
+  let mockComparisonCache: ComparisonCache
+  beforeEach(() => {
+    mockComparisonCache = new ComparisonCache()
```

Comment:
:art: a _mock_ object is one that simulates the behaviour of a real method or object. This looks like you're using the same implementation, so could this just be named `comparisonCache`?